common-compose.yml

```diff
@@ -0,0 +1,70 @@
+wordpress:
```

Comment:
Still working on merging the `common-compose.yml` into the `docker-compose.yml`?